hooks/environment.agent

```diff
@@ -134,6 +134,10 @@ function get_initial_job_id() {
         echo "ERROR: Initial job ID output invalid:\n${CURL_OUTPUT}" >&2
         echo "Retrying up to $((3 - $idx)) more times before failing out..." >&2
     done
+    # If we fail out, let's print the curl output to the log:
+    echo "Begin curl output:"
+    curl --fail-with-body -L -H "${TOKEN_HEADER}" "${URL}"
```

Comment:
```suggestion
    curl --fail-with-body -vL -H "${TOKEN_HEADER}" "${URL}"
```

Let's be verbose, no need to be shy

Reply:
Done (in https://github.com/JuliaCI/cryptic-buildkite-plugin/pull/40/commits/a45dd71518349150fa9d4fae60c03479f796c661).